packages/core/helper-plugin/babel.config.js

```diff
@@ -1,21 +0,0 @@
-const defaultPresets = [
-  [
-    '@babel/preset-env',
-    {
-      modules: 'commonjs',
-      loose: true,
-    },
-  ],
-];
-
-module.exports = {
-  presets: defaultPresets.concat(['@babel/preset-react', '@babel/preset-flow']),
-  plugins: [
-    ['@babel/plugin-proposal-class-properties', { loose: true }],
-    ['@babel/plugin-proposal-private-property-in-object', { loose: true }],
-    ['@babel/plugin-proposal-private-methods', { loose: true }],
```

Comment:
I guess those are the plugins that are no longer needed and that you mentioned in your PR description?

Reply:
Yes. They are all part of `@babel/preset-env` now.